mappings/net/minecraft/entity/player/PlayerInventory.mapping

```diff
@@ -19,10 +19,12 @@ CLASS net/minecraft/class_1661 net/minecraft/entity/player/PlayerInventory
 		ARG 2 stack
 	METHOD method_7368 getHotbarSize ()I
 	METHOD method_7369 clearItem (Ljava/util/function/Predicate;I)I
-		ARG 1 itemPredicate
+		ARG 1 predicate
 		ARG 2 amount
 	METHOD method_7370 getBlockBreakingSpeed (Lnet/minecraft/class_2680;)F
 		ARG 1 block
+	METHOD method_7371 getIndexOf (Lnet/minecraft/class_1799;)I
```

Comment:
```suggestion
	METHOD method_7371 index (Lnet/minecraft/class_1799;)I
```

Reply:
Lists and stuff usually call this indexOf, no?

Reply:
sure, but `index` itself is a transitive verb, without a proposition. i.e. "index this item stack in this inventory"